recipes/pyorc/meta.yaml

```
@@ -0,0 +1,55 @@
+package:
+  name: pyorc
+  version: "0.4.0"
+
+source:
+  url: https://pypi.io/packages/source/p/pyorc/pyorc-0.4.0.tar.gz
+  sha256: af56c50dd0545b451a37d38e73e02405f332e91779cf9ab1c8ae3e61b9eb0c90
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: |
+    {{ PYTHON }} setup.py build_orc --build-type=release
+    {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
+  host:
+    - python
+    - pip
+    - pybind11
```

There might be others but it looks like the orc build needs some dependencies adding:
```suggestion
    - pybind11
    - snappy
    - zstd
    - zlib
```

---

It's going to be something else. These libraries should be built from source during the ORC C++ package install. I'll try to look into it why the snappy build fails.

---

Could it be possible to just use the orc package instead of building it here?

---

I don't think that the ORC package comes with a binary distribution.

---

Every conda package is a binary package. `conda install orc` gives you a compiled Apache ORC C++

---

Thank you, did not know that. Managed to build the python module.